### ISSUE TITLE
I accidentally unlocked my search results window and lost it. How do I bring my search results window back and lock it back in at the bottom?

### ISSUE THREAD

**Issue by aminaahmad46** [notepad-plus-plus/notepad-plus-plus#5167]
### Description of the Issue

I accidentally unlocked my search results window and lost it. How do I bring my search results window back and lock it back in at the bottom?

### Steps to Reproduce the Issue

1. Somehow, after completing a Find in Files search, I clicked in the results window and unlocked it from the bottom of my screen. 
2. In an attempt to lock it back to the bottom of my screen, it disappeared into the abyss of my screen.

I've looked under all the menu options and am unable to find where I can have the search results window come back up even when running more searches. It completes the search but I cannot seem to get the results window back in my screen where I can see it.

### Actual Behavior

I was able to get back the results window by detaching from my second screen but I would still like to lock it back into my Notepadd++ screen at the bottom. The "restore" is disabled and the other options do not lock it back into the screen. 

![image](https://user-images.githubusercontent.com/39168206/50456164-83131e00-0907-11e9-944b-b769a2f8dc78.png)

**Comment by gurikbal** [comment]
mouse point is ready there
double click at Find result - 0 hits bar

**Comment by aminaahmad46** [comment]
Thank you but double clicking on the Find result is how I unlocked it from the window. Double clicking on it again, maximizes the find result window to full screen. 

I had to detach my second monitor and complete a search again after that for it to appear on my screen. At Notepadd++ maximized window, I dragged the Find results window to the bottom of Notepad++ screen (the borderline of Notepad++) and then that created an outline of the Find results window at the bottom, I let go of the drag and dropped it there which then the Find result successfully locked the window back to the bottom. I should've figured that out since other programs run the same way. 

Thanks though.
